Documentation/README_RiggedHandVisualizer.md

```diff
@@ -1,4 +1,4 @@
-# Rigged Hand Visualizer [Experimental]
+# Rigged Hand Visualizer
```

Comment:
It would be awesome if we could make this documentation more tutorial-esque, with screen shots of the inspector and property descriptions, but that would probably be a different PR.